etc/SuperBuild/cmake/Modules/BuildZLIB.cmake

```diff
@@ -10,8 +8,7 @@ set(ZLIB_ARGS
 ExternalProject_Add(
     ZLIB
     PREFIX ${CMAKE_CURRENT_BINARY_DIR}/ZLIB
-    GIT_REPOSITORY ${ZLIB_GIT_REPOSITORY}
-    GIT_TAG ${ZLIB_GIT_TAG}
+    URL "https://zlib.net/fossils/zlib-1.2.11.tar.gz"  # Original repository
```

Comment:
Why is this change included?

Reply:
It is unrelated, but I noticed you were picking ZLIB from a, afaik, non-official distribution.  Also, with that you need to update the tag on every release.  By pointing to the official distro of directory fossils, you can pick any version and update it whenever you see fit.

Reply:
The github repository is official, from the zlib home page:
"zlib development contributions and issues are best addressed on [zlib's GitHub repository](https://github.com/madler/zlib)"